app/views/departments/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Department New</h1>
+
+<%= form_with(model: @department, url: departments_path) do |f| %>
+    <%= f.label :name %>
+    <%= f.text_field :name %>
+
+    <%= f.label :abbreviation %>
```

Comment:
Also, keep this in mind. The part inside the form for `new` and `edit` views are the same, so use partial for it.